conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
```

Comment:
If you're concerned about combining the files, just rename `conda/core/repodata.py` to `conda/core/repodata_impl.py`